src/main/java/com/assignment/StudentInfo.java

```diff
@@ -0,0 +1,96 @@
+package com.assignment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+
+public class StudentInfo {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    ArrayList<Student> studentList = new ArrayList<>();
+
+    public void setStudentInfo(String[] info) {
+        Student student = new Student();
+
+        student.setId(info[0]);
+        student.setName(info[1]);
+        student.setAddress(info[2]);
+        student.setStudentClass(info[3]);
+        student.setSchool(info[4]);
+        studentList.add(student);
+        logger.info("Student information added");
+    }
+
+    public Student[] getStudentList() {
```

Comment:
Instead of returning an array of strings, why don't we return a list of Student objects?